pom.xml

```diff
@@ -1788,6 +1788,23 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>jdk17</id>
```

Comment:
Rather than activating manually, you can active automatically based on JDK version:
```xml
<activation>
  <jdk>[17,)</jdk>
</activation>